appveyor.yml

```diff
@@ -123,6 +127,32 @@ branches:
   only:
     - auto
 
+after_test:
```

Comment:
Oh wait right we can't do that because otherwise the `artifacts` listing on AppVeyor fails. Those artifacts have to be in place before the deploy (and before_deploy) step starts I believe.